src/plugins/vis_type_vislib/public/fixtures/_vis_fixture.js

```diff
@@ -20,13 +20,20 @@
 import _ from 'lodash';
```

Comment:
It seems like most of this file is only used in the legacy tests, can we move it over there? The only thing used in NP jest tests as well is the `getMockUiState` function, looks like it makes sense to move that one into a separate file so we don't have to import things deeply from within other plugins and mess with jquery in the new platform.

Reply:
Ok, done